### ISSUE TITLE
[Bug] Multiple issues

### ISSUE THREAD

**Issue by KostyaEver** [ever-co/ever-teams#1436]
- [x] <del>Task details | User should be able to select / change 'Epic' for type of issues such as 'Story', 'Task', 'Bug'. (Now it works only for Story)<del>

<img width="393" alt="Screenshot at Oct 02 09-53-01" src="https://github.com/ever-co/ever-teams/assets/107203576/f2b918ff-a502-4373-89b2-8ccd2bb8d930">
<img width="386" alt="Screenshot at Oct 02 09-53-34" src="https://github.com/ever-co/ever-teams/assets/107203576/7a64abd9-726e-4383-8359-eee5a81a6b84">

- [x] Task details | Task Header - Next to exact task should be reserved space for 'Epic' icon OR 'Select Epic'. Only after it should go 'add parent' OR 'change parent' (if there is a parent issue); Note: Epic is different a bit - as a more global parent. 'Issue' parent can be Story, Task, or Bug.
Example: #issue number | Issue name | Epic | any 'parent issue' or button 'add parent'
Color/icons should be the same for all types as defined 

<img width="1550" alt="Screenshot at Oct 02 09-33-28" src="https://github.com/ever-co/ever-teams/assets/107203576/13851927-dcd7-4cf1-8c50-5399317a6b79">

- [x] Task details | Parent logic - if issue 1 is a parent for issue 2, but issue 2 is the parent for issue 3, issue 3 is the parent for issue 4, and so on. Most important that issue 1 can not be a child of issues 2, 3, 4 and so on as well as issue 2 can not be a child for issue 4 and so on.
In another case, issue 1 can be a parent for 2,3,4 directly. 
**Note**: one issue has only one parent!!!! but can have a multiple child issues

- [x] Team Settings | When a user is creating a Status, Lables, Vesions and so on with suggested 'icons', Should NOT see duplicates of icons. should see only unique icons.

<img width="955" alt="Screenshot at Sep 25 14-44-29" src="https://github.com/ever-co/ever-teams/assets/107203576/fad22768-73b3-453a-86ac-5620c808eac9">

- [x]  User's Task page | When Manage is creating a new task on the teammate's page, there is no option to select 'issue type' that makes the creator go to another page to set a type. Now it creates just default type.

<img width="1355" alt="Screenshot at Sep 25 15-03-09" src="https://github.com/ever-co/ever-teams/assets/107203576/c7a7951c-2714-438b-9f1b-81bcce94ce66">

- [x] Page 'Task Details' right side menu 'Status' block - Versions (item) should be displayed just selected ' icon' next to the title 'Versions'. (We do not have icons for version)

<img width="302" alt="Screenshot at Sep 25 16-57-37" src="https://github.com/ever-co/ever-teams/assets/107203576/f68ec0b9-b9d6-4efa-824a-8b52175f1d37">

- [x] Page 'Task Details' arrow button 'back' does not work (Working)

<img width="773" alt="Screenshot at Sep 25 15-45-52" src="https://github.com/ever-co/ever-teams/assets/107203576/bf0cb19d-4901-4274-b4eb-52ff5ab1fdde">

- [x] Page 'User Tasks' does not display in column 'Worked Today' (Today and Total time values) for Tabs Worked / Assigned / Unassigned, since on the main page it displays very well

<img width="1334" alt="Screenshot at Sep 25 15-54-45" src="https://github.com/ever-co/ever-teams/assets/107203576/f2849cc3-039b-44ed-884b-6397fed2fffd">

- [x] Page 'Team page' incorrectly displays task 1 of another teammate, since the teammate is tracking time for a specific task 1, for another user it is displaying incorrectly (task 4)

<img width="1885" alt="Screenshot at Sep 25 16-04-28" src="https://github.com/ever-co/ever-teams/assets/107203576/cfe3ea72-9162-4507-bc38-2e6548b5681a">

- [x] Page 'Task Details' in Task Description 'Text Editor' option to add a link to any word in a text is not working. (In Safari)

<img width="611" alt="Screenshot at Sep 25 16-19-28" src="https://github.com/ever-co/ever-teams/assets/107203576/4c556b58-b031-422f-bf72-dde0f902a83c">

- [x] Page 'Task Details' in Task Descriptio, the button 'Cancel' does not work as expected. Should not save the latest changes in the task description.

<img width="1173" alt="Screenshot at Sep 25 16-27-23" src="https://github.com/ever-co/ever-teams/assets/107203576/712820ee-e019-48b3-beb2-33f5bc3e529f">

- [x] Page 'Team page' | Issue 'Type' selector popup '+ New Issue', let's hide the button for now

<img width="1285" alt="Screenshot at Sep 25 16-50-59" src="https://github.com/ever-co/ever-teams/assets/107203576/5fdfee9a-93ac-4e43-bdde-c93bd36fa434">

- [x] Parent / Child task - Tasks can not be Parent for each other. Child issue has ONLY one Parent. Parents may have multiple Child issues. For now, if Child Issue 1 has Parent so issue 1 can not be paren for other issues.

<img width="1469" alt="Screenshot at Sep 25 17-22-31" src="https://github.com/ever-co/ever-teams/assets/107203576/55096484-7093-4b31-aab4-99d5447665b0">

- [x] Page 'Task Details' | Status block - when 'Epic' is selected, there is no icon and appropriate color.

<img width="411" alt="Screenshot at Sep 25 17-44-58" src="https://github.com/ever-co/ever-teams/assets/107203576/3ce971fe-a2aa-42d3-8a52-5a851d81535c">

- [x] Issue Relations | One Detailed Task page I can see a dropdown with different relations AND I don't see in Settings pre-entered items. Where are these items? Where users can create/remove more?

<img width="1397" alt="Screenshot at Sep 25 17-57-48" src="https://github.com/ever-co/ever-teams/assets/107203576/961f1d77-6c41-4c89-bad9-289f51fe5a5f">
<img width="946" alt="Screenshot at Sep 25 17-58-37" src="https://github.com/ever-co/ever-teams/assets/107203576/69d42516-25fe-4b1a-8d6d-30e837b89e13">

**Comment by desperado1802** [comment]
@KostyaEver the hyper-link functionality should work
https://github.com/ever-co/ever-teams/assets/124465103/09934ad7-ab61-4d18-a4c2-a5b6f4666d1a

**Comment by badalkhatri0924** [comment]
1. TASK|BUG can have ParentId to be set either to Story ID or Epic ID
2. Story can have ParentId set to Epic ID
3. Epic can't have ParentId set
4. ParentId field is nullable, i.e. issue may not have any parents at all (e.g. I can create Bug that not assigned to any Epic or Story) | (same I can create Task that is not assigned to any Story / EPic)
5. In below cases, we can show "read-only" Epic name in the Sidebar: If current issue is BUG / TASK type and Parent of it is Story that belong to Epic
